### PR TITLE
feat(launch): initialize request pool

### PR DESCRIPTION
**Pull request by lubtd**
Request pool is a list of the different request for a specific chain (add account, add validator, ...)

@fadeev this is an example of an "indexed list"

TODO:

- [x] Implement tests for genesis validation
- [x] Edit `list` request to list the request from a chain id

### CODE REVIEW

File: x/launch/client/cli/query_request_test.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
+package cli_test
+
+import (
+	"fmt"
+	"github.com/tendermint/spn/testutil/sample"
```
Comment (Pantani) [inline x/launch/client/cli/query_request_test.go:5]: import sort does not respect the lint style

File: x/launch/keeper/request_test.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
+package keeper
+
+import (
+	"github.com/tendermint/spn/testutil/sample"
```
Comment (Pantani) [inline x/launch/keeper/request_test.go:4]: import sort does not respect the lint style

**Review by Pantani — APPROVED** [review]

**Review by fadeev — DISMISSED** [review]
👍

File: x/launch/genesis.go [hunk 3]
```diff
@@ -28,6 +39,14 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 
 	// this line is used by starport scaffolding # genesis/module/export
+
+	// Get all request
+	requestList := k.GetAllRequest(ctx)
+	for _, elem := range requestList {
+		elem := elem
```
Comment (ilgooz) [inline x/launch/genesis.go:46]:
```suggestion
```

**Review by ilgooz — APPROVED** [review]

File: x/launch/genesis.go (hunk 3 above)
Comment (lubtd) [reply x/launch/genesis.go:46]: https://github.com/tendermint/spn/pull/179#discussion_r674209069

Comment (ilgooz) [reply x/launch/genesis.go:51]: Please check https://github.com/tendermint/spn/pull/179#discussion_r677302544.

Comment (lubtd) [reply x/launch/genesis.go:51]: Good observation.
I created a PR to look at this change once all the structures for the milestone 1 are initialized

**Review by ilgooz — APPROVED** [review]

**Review by Pantani — APPROVED** [review]

**Pull request merged**